lib/ain-grpc/src/call_request.rs

```diff
@@ -39,6 +39,27 @@ pub struct CallRequest {
 }
 
 impl CallRequest {
+    pub fn guess_tx_type(&self) -> Result<Self, Error> {
```

Comment:
Let's rename this. The response does co-relate well with the guess name. 

Perhaps `clone_with_guessed_tx_type`

Reply:
Or better yet - even better is `guess_tx_type` simply just returns the guess. 

And then do a clone internally in the method.

Reply:
Renamed to `clone_with_guessed_tx_type` for now. We can decide on how to properly handle the mutation after we have unblocked the RPC users.